p2p/src/authenticated/actors/tracker/record.rs

```diff
@@ -1,227 +1,521 @@
 use crate::authenticated::types::PeerInfo;
 use commonware_cryptography::Verifier;
 use std::net::SocketAddr;
+use tracing::trace;
 
 /// Represents information known about a peer's address.
-#[derive(Clone)]
-pub enum Record<C: Verifier> {
+#[derive(Clone, Debug, PartialEq)]
+pub enum Address<C: Verifier> {
     /// Peer address is not yet known.
     /// Can be upgraded to `Discovered`.
-    /// Tracks the number of peer sets this peer is part of.
-    Unknown(usize),
+    Unknown,
+
+    /// Peer is the local node.
+    LocalNode(PeerInfo<C>),
```

Comment:
Thought about this some more. I wonder if `Me` (preferred to `Myself`) or `Identity` would be clearer here? I wish we could use `Self` but alas...

I'm now against using `Local*` to avoid IP confusion (although I previously suggested).

As soon as this is addressed, should be good to merge.